packit.yaml

```diff
@@ -0,0 +1,35 @@
+upstream_project_url: https://github.com/rhinstaller/anaconda-webui
+# enable notification of failed downstream jobs as issues
+issue_repository: https://github.com/rhinstaller/anaconda-webui
+specfile_path: anaconda-webui.spec
+upstream_package_name: anaconda-webui
+downstream_package_name: anaconda-webui
+# use the nicely formatted release description from our upstream release, instead of git shortlog
+copy_upstream_release_description: true
+
+srpm_build_deps:
+- npm
+- make
+
+actions:
+  post-upstream-clone: make anaconda-webui.spec
+  create-archive: make dist
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+    - fedora-development
+
+  - job: copr_build
+    trigger: release
+    owner: "@rhinstaller"
+    project: "Anaconda-webui"
+    preserve_project: True
+    actions:
+      post-upstream-clone: make anaconda-webui.spec
+      # HACK: tarball for releases (copr_build, koji, etc.), copying spec's Source0; this
+      # really should be the default, see https://github.com/packit/packit-service/issues/1505
+      create-archive:
+        - sh -exc "curl -L -O https://github.com/rhinstaller/anaconda-webui/releases/download/${PACKIT_PROJECT_VERSION}/${PACKIT_PROJECT_NAME_VERSION}.tar.xz"
+        - sh -exc "ls ${PACKIT_PROJECT_NAME_VERSION}.tar.xz"
```

Comment:
I think we want to have nightly builds from HEAD?

Reply:
Sure, we can extend packit configuration with anything extra as a followup.